### PR TITLE
[Fix] Fix loss parse in val_step

### PR DESCRIPTION
**Pull request by acdart**
## Motivation

Add loss parse in val_step, so that logger can get loss value of validation dataset
 
## Modification

add loss parse and format output dict in val_step function

### CODE REVIEW

**Comment by Junjun2016** [comment]
Hi @acdart 
Thanks for your contribution.
It would be better to add some unittests if your time is available.

File: tests/test_models/test_segmentors/utils.py [hunk 1]
```diff
@@ -101,6 +101,14 @@ def _segmentor_forward_train_test(segmentor):
         imgs, img_metas, gt_semantic_seg=gt_semantic_seg, return_loss=True)
     assert isinstance(losses, dict)
 
+    # Test val_step
+    with torch.no_grad():
+        segmentor.eval()
+        data_batch = dict(
+            img=imgs, img_metas=img_metas, gt_semantic_seg=gt_semantic_seg)
+        outputs = segmentor.val_step(data_batch)
+        assert isinstance(outputs, dict)
```
Comment (Junjun2016) [inline tests/test_models/test_segmentors/utils.py:110]: Can also add asserts for the `keys` in `outputs`.

**Review by Junjun2016 — APPROVED** [review]
LGTM
